doc/Examples/openroad-sky130.md

```diff
@@ -0,0 +1,94 @@
+# Introduction with Sky130 and OpenROAD
+
+The following directions will get a simple ``pass`` design from RTL to GDS using the [OpenROAD tools](https://theopenroadproject.org) and the [Skywater 130nm PDK](https://github.com/google/skywater-pdk). These directions are meant to provide the minimal set of steps to do so, please reference the next section, [Hammer End-to-End Integration Tests](https://hammer-vlsi.readthedocs.io/en/stable/Examples/e2e.html), for more detailed descriptions of all files and commands.
```

Comment:
Is it not possible to link to other chapters by reference instead of URL because this is markdown and not RST?

Reply:
Yep exactly, as far as I can tell.